serde-generate/src/golang.rs

```diff
@@ -345,11 +344,11 @@ for _, item := range(value) {{
                 write!(
                     self.out,
                     r#"
-if err := serializer.SerializeLen(len(value)); err != nil {{ return err }}
+if err := serializer.SerializeLen(uint64(len(value))); err != nil {{ return err }}
 offsets := make([]int, len(value))
 count := 0
```

Comment:
Can you make `offsets` an array of uint64 instead of casting below?

Reply:
SortMapEntries(offsets []int), so offsets should be []int, unless you want to change this interface too